### DISCUSSION TITLE
Organization Roles are too limited - missing "Trusted Member" role.

### DISCUSSION THREAD

**Discussion by nizzantalon** [community/community#60519] [Code Security]
### Select Topic Area

Product Feedback

### Body

Many features are only accessible today to the organization Owner(s) which should generally be available to a larger group of trusted members. These members don't necessarily need access to organization billing, admin all repositories, or manage members.

The features I'd like them to access, at least to some extent: 
1. Organization-level secrets and variables
2. Self-hosted Runners - both setting up new runners and seeing the current runner status (current job in progress)
3. Webhooks

Being able to assign such a role to DevOps/Automation engineers working on Actions would greatly reduce risk of assigning them as organization owners.

**Comment by drahamim** [comment]
In addition being able to grant only specific users ability to create repositories without granting them full admin rights to an org would also be useful.
